.github/workflows/versions-propagation-auto-merge.yml

```diff
@@ -1,5 +1,10 @@
 name: Versions propagation auto-merge
-on: pull_request
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
```

Comment:
```suggestion
    types: [opened, reopened, labeled]
    branches:
      - develop
```

Reply:
On second thought, it might be better not to have all types on one line, as you wish.
Don't we want to restrict to PRs for develop branch though?

Reply:
We also need it for maintenance branches (7.4.x, for instance), but maybe it's better to keep `develop` here and update the branch name directly there.